presto-spi/src/test/java/io/prestosql/spi/type/TestTimestamps.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.spi.type;
+
+import org.testng.annotations.Test;
+
+import static io.prestosql.spi.type.Timestamps.roundDiv;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestTimestamps
+{
+    @Test
+    public void testRoundDiv()
```

Comment:
perhaps add a test with 0 divisor (failure)
it would be great if negative divisor was rejected too